tests/samples_tests/smoke_tests/test_benchmark_app.py

```diff
@@ -40,7 +51,8 @@ def verify(sample_language, device, api=None, nireq=None, shape=None, data_shape
         *('-api', api) if api else '',
         *('-dump_config', tmp_path / 'conf.json') if tmp_path else '',
         *('-exec_graph_path', tmp_path / 'exec_graph.xml') if tmp_path else '',
-        '-d', device, '-b', '1', '-niter', '10'
+        *('-b', batch) if batch else '',
+        '-d', device, '-niter', '10'
```

Comment:
`Windows (VS 2019, Python 3.11) / Samples (pull_request)`
```
  File "C:\actions-runner\_work\openvino\openvino\install\python\openvino\tools\benchmark\benchmark.py", line 168, in main_loop
    fps = len(batch_size) * iteration / total_duration_sec
          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~
ZeroDivisionError: float division by zero
```
fails because `benchmark_app` exits too quickly. One solution is to fix the `benchmark_app`, or you can increase `-niter` for your test so it takes more time.

Reply:
Add '-t' to control execute time.